user/pages/03.Howtos/02.openstack-cli/docs.de.md

```diff
@@ -140,22 +136,32 @@ openstack --help
 
 ### Plugins installieren
 
-Es besteht die Möglichkeit Plugins zu installieren. Dafür fügt man in den nachfolgenden Befehl einfach das entsprechende Plugin ein:
+Es besteht die Möglichkeit, Plugins zu installieren. Dafür fügt man in den nachfolgenden Befehl einfach das entsprechende Plugin ein:
 
 ```shell
 pip install python-<PLUGINNAME>client
 ```
 
-Plugins können dann wie folgt installiert werden:
+Benötigte Plugins für den OpenStack (automatisch als Abhängigkeit installiert):
 
-```shell
-pip install python-heatclient
-```
+* keystone - Identity API
+* nova - Compute API
+* cinder - Block Storage Volume API
+* glance - Image API
```

Comment:
for me glanceclient was not installed automatically as a dependency, maybe better move it down to "zusätzlich installiert"

Reply:
On my system, this was a dependeny (in the Required: line):
```
$ pip show python-openstackclient
Name: python-openstackclient
Version: 4.0.0
Summary: OpenStack Command-line Client
Home-page: https://docs.openstack.org/python-openstackclient/latest/
Author: OpenStack
Author-email: openstack-discuss@lists.openstack.org
License: UNKNOWN
Location: /Users/mbernhardt/.pyenv/versions/3.7.5/lib/python3.7/site-packages
Requires: python-glanceclient, cliff, python-novaclient, six, osc-lib, openstacksdk, oslo.i18n, Babel, oslo.utils, python-keystoneclient, pbr, keystoneauth1, python-cinderclient
Required-by: python-octaviaclient
```

Reply:
I wasn't able to clarify this, so I tried to put it differently.